NetKAN/Karbonite.netkan

```diff
@@ -26,7 +27,7 @@
         {
             "file"       : "GameData/UmbraSpaceIndustries",
```

Comment:
"find": "UmbraSpaceIndustries/Karbonite",
"install_to" : "GameData/UmbraSpaceIndustries"

No filter required (means that changes to USI-Core will not break the Karbonite install.